src/Test/Perf/tests/csharp/csharp_compiler.csx

```diff
@@ -1,6 +1,6 @@
 // Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
 
-#load "../../../util/test_util.csx"
+#load "../../util/test_util.csx"
```

Comment:
I'm both surprised and thrilled that this works... :smile:  Perhaps we should switch all ```#loads``` to '/' (assuming the the corefx API on Linux will not be so forgiving if they're all '\')?